src/api/manager/resolvers.js

```diff
@@ -561,9 +575,167 @@ const resolvers = {
         console.log(e)
       }
     },
+    addMultiRecords: async (_, { name, records }, { cache }) => {
+      const ens = getENS()
+
+      function setupTransactions({ name, records, resolverInstance }) {
+        try {
+          const resolver = resolverInstance.interface.functions
+          const namehash = getNamehash(name)
+          console.log('**** records', records)
+          const transactionArray = records.map((record, i) => {
+            switch (i) {
+              case 0:
+                if (!record) return undefined
+                let encoded = resolver['setAddr(bytes32,address)'].encode([
+                  namehash,
+                  record
+                ])
+                return encoded
+              case 1:
+                if (!record) return undefined
```

Comment:
Same here

Reply:
Not sure this works as the other 2 cases return empty arrays not undefined AFAIK. I'm going to leave this for now just in case